kubectl-plugin/pkg/cmd/ray.go

```diff
@@ -0,0 +1,26 @@
+package cmd
+
+import (
+	cluster "github.com/ray-project/kuberay/kubectl-plugin/pkg/cmd/cluster"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	CLIName = "ray"
+)
+
+func NewRayCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "ray",
+		Short:        "ray kubectl plugin",
+		Long:         `Manage RayCluster resources.`,
```

Comment:
why do we use backtick instead of `"`?

Reply:
It was originally multiple lines. Changed to `"`